index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+      integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Portfolio Setup and Mobile first</title>
+  </head>
+  <body>
+    <div class="logo-menu">
+      <div class="logo" onclick="window.location.href='index.html'">
+        <p class="logo">My Logo</p>
+      </div>
+      <i class="fa fa-bars"></i>
+    </div>
+    <main>
+      <div class="main-container">
+        <div class="title">
+          <h2>I’m Mariane Glad to see you!</h2>
+        </div>
+        <div class="titleText">
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+        </div>
+        <div class="connect">
+          <p>LET'S CONNECT</p>
```

Comment:
- [ ] Kindly I recommend you to use uses a header tag `hx` (examples: `h2,h3,` etc...). This way will be semantically correct and the screen readers and other technologies will thank you.